src/us/kbase/narrativejobservice/test/AweClientDockerJobScriptTest.java

```diff
@@ -495,7 +498,7 @@ public void testBasicProvenance() throws Exception {
         checkLoggingComplete(res);
         
         release = "beta";
-        ver = "0.0.8";
+        ver = "0.0.7";
```

Comment:
@scanon: because 8-th version appeared in auth2ci environment, in CI it's still 7-th.